notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def regognize_audio(compiled_asr_model, src_audio):
</pre><p>regognize -&gt; recognize</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>also it is good for demonstration result provide also reference and add audio player with input audio that user can check quality of the result </p>

Reply:
Done